### DISCUSSION TITLE
--installpkg Ignores deps, intended and how to work around

### DISCUSSION THREAD

**Discussion by tapetersen** [tox-dev/tox#2263] [Q&A] [answered]
Hello, I'm using tox for running tests in a project where I have a makefile that builds packages separately when making a release.
I would like to test those using tox without rebuilding them a second time but instead install from the already built package. 

Using the `--installpkg` option seems intended for this usage but it adds the `--no-deps` flag to pip  install which means its requirements aren't installed. I could of course add those to the tox.ini as well but I would rather keep that information in the setup.cfg only.

I can understand why you sometime would like to explicitly only install a and not its deps it would be useful if the alternative exists and would actually be the default I would've expected. Is there some way to get the desired behavior anyway either by telling tox to install dependencies from setup.cfg or installing from the built package including deps?

**Comment by gaborbernat** [comment, marked as answer]
This improvement for ``--installpkg`` has been added to tox 4 via https://github.com/tox-dev/tox/pull/2235. There are no plans currently to support this within tox 3.

**Reply by tapetersen** [reply]
Ok I'll try and see if tox4 works for us as we're not depending on any external plugins. Thanks for developing tox and taking time to help.
